app/models/product.rb

```diff
@@ -15,6 +15,9 @@ class Product < ActiveRecord::Base
   has_many   :price_policies
   has_many   :training_requests, dependent: :destroy
 
+  cattr_accessor(:types) { ConfigurableArray.new([Instrument, Item, Service, Bundle]) }
+  cattr_accessor(:mergeable_types) { ConfigurableArray.new([Instrument, Item, Service, Bundle]) }
```

Comment:
`types` is such a generic name that adding it to the class_variable list feels a bit scary, especially since we already have a class hierarchy under `Product` that will result in `Item.types`, etc appearing. Should we change to `product_types` at least? Or even stick with the class instance variables we had previously since this isn't saving that many lines?